src/renderer/components/watch-video-live-chat/watch-video-live-chat.css

```diff
@@ -22,6 +22,10 @@
   row-gap: 16px;
 }
 
+.watchingCount {
+  font-weight: normal;
```

Comment:
**suggestion (non-blocking):** Add space between title text and count. Decrease font size of count and use tertiary color to better direct focus.

```suggestion
  font-weight: normal;
  margin-inline-start: 5px;
  font-size: 15px;
  color: var(--tertiary-text-color);
```

![Screenshot_20231116_204817](https://github.com/FreeTubeApp/FreeTube/assets/84899178/48a9d693-39bb-4c50-98fa-1a0134467d63)